x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/risk_score/index.ts

```diff
@@ -27,6 +27,14 @@ export interface HostsRiskScore {
   host: {
     name: string;
   };
-  risk_score: number;
   risk: string;
+  risk_stats: {
```

Comment:
@ecezalp I am updating the `HostsRiskScore` interface to match the new version of the transform.

Reply:
I am a bit unsure about the implications - what would happen if someone installed the host risk score module during 7.16 and then upgraded to 8? Do we have a mechanism to tell them that the also need to update the host risk score package? How would they know? Would they get a broken UI?

Maybe we don't care at this stage about breaking changes because the feature is experimental, but I think it's still something we should clear with product, what happens if users have an older version of the host risk score package installed. What issue would the users encounter, and how would they know how to fix it?

Reply:
@ajosh0504 ⬆️